src/Devito.jl

```diff
@@ -37,6 +37,52 @@ end
 configuration(key) = PyDict(devito."configuration")[key]
 configuration() = PyDict(devito."configuration")
 
+function configure_nvidia_hpcsdk(version::String)
+
+    version === "22.3" || throw(ArgumentError("Only Nvidia HPC_SDK version 22.3 is currently supported!") )
+
+    # HPC-SDK location 
+    HPCSDK_HOME  = "/opt/nvidia/hpc_sdk/Linux_x86_64/2022"
```

Comment:
should this be hard-coded?

Reply:
I think so, for version 22.3 specifically